lib/tasks/advanced_search.rake

```diff
@@ -0,0 +1,11 @@
+require "publishing_api_finder_publisher"
+
+namespace :publishing_api do
```

Comment:
Would be good to add a schema test for the payload here too.

Reply:
Added here: https://github.com/alphagov/rummager/pull/1196/commits/c8a2b2d587ee5af9059968cdc52de207fe244f1e#diff-794b684fdca56d301fb5ed3d322928e2R89 this fails if I change the advanced search data to be `document_type: search` though.
Errors: `The property '#/details' contains additional properties [\"document_noun\", \"show_summaries\", \"f...` Perhaps this is how I'm configuring the schema test helper?